extension-config.json

```diff
@@ -1,5 +1,5 @@
 {
-  "version": "2.2.0",
+  "version": "2.2.2-rc.4",
```

Comment:
hm, don't want to release this to master, if we are releasing then we need to actual version. That said, it's weird that we skipped `2.2.1`?